docs/metrics.md

```diff
@@ -0,0 +1,214 @@
+
+For example, statistics of the inference execution queue, model runtime parameters etc. They can also track the usage based on model version, API type or requested endpoint methods.
+
+OpenVINO Model Server metrics are compatible with Prometheus standard [Prometheus metrics detailed description](https://prometheus.io/docs)
+
+They are exposed on the `/metrics` endpoint.
+
+## Available metrics families
+
+Metrics from default list are enabled with the `metrics_enabled` flag or json configuration.
+
+However, you can enable also additional metrics by listing all the metrics you want to enable in the `metric_list` flag or json configuration.
+
+
+Default metrics
+| Type      | Name | Labels | Description |
+| :---    |    :----   |    :----   |    :----       |
+| gauge      | ovms_streams | name,version | Number of OpenVINO execution streams |
+| gauge      | ovms_current_requests | api,interface,method,name,version | Number of inference requests currently in process |
+| counter      | ovms_requests_success | name,version | Number of successful requests to a model or a DAG. |
+| counter      | ovms_requests_fail | name,version | Number of failed requests to a model or a DAG. |
+| histogram      | ovms_request_time_us | interface,name,version | Processing time of requests to a model or a DAG. |
+| histogram      | ovms_inference_time_us | name,version | Inference execution time in the OpenVINO backend. |
+| histogram      | ovms_wait_for_infer_req_time_us | name,version | Request waiting time in the scheduling queue. |
+
+Optional metrics
+| Type      | Name | Labels | Description |
+| :---    |    :----   |    :----   |    :----       |
+| gauge      | ovms_infer_req_queue_size | name,version | Inference request queue size (nireq). |
+| gauge      | ovms_infer_req_active | ,name,version | Number of currently consumed inference request from the processing queue. |
+
+Labels description
+| Name      | Values |  Description |
+| :---    |    :----   |    :----   |
+| api      | KServe, TensorFlowServing  | Name of the serving API. |
+| interface      | REST, gRPC | Name of the serving interface. |
+| method      | ModelMetadata, ModelReady, ModelInfer, Predict, GetModelStatus, GetModelMetadata | Interface methods. |
+| version      | 1, 2, ..., n | Model version. Note that GetModelStatus and ModelReady do not have the version label. |
+| name      | resnet, face_detection, ... | Model name. |
+
+
+## Enable metrics
+
+By default, the metrics feature is disabled.
+
+Metrics endpoint is using the same port as the REST interface for running the model queries.
+
+It is required to enable REST in the model server by setting the parameter --rest_port.
+
+To enable default metrics set you need to specify the `metrics_enabled` flag or json setting:
+
+CLI
+
+   ```bash
+         docker run --rm -d -v ${PWD}/models/resnet-50-tf:/opt/model -p 9001:9001 -p 9002:9002 openvino/model_server:latest \
```

Comment:
while there is added a command to mount a folder, there should be added a command to create the model repository or to use a public model from gc. Use the same method in all examples

Reply:
Ok.